frontend/src/_layouts/base.html

```diff
@@ -113,6 +117,14 @@
       </div>
     </footer>
     <!-- Footer -->
+
+    <div
+      x-data="{ open: false }"
+      @toggle-backdrop.window="open = $event.detail"
+      x-show="open"
+      class="fixed h-full w-full bg-white/50 backdrop-blur-sm md:hidden md:open:hidden"
```

Comment:
See how `fixed z-10 inset-0 opacity-50 backdrop-blur-sm` works.

Few points here:
- `inset-0` instead of `h-full w-full`. It worked for the previous incarnation of the backdrop.
- Plain `opacity` instead of the background opacity.
- No need for a special treatment for `md` screen size, I guess.

Reply:
`inset-0` instead `h-full w-full`